js/scripts.js

```diff
@@ -459,4 +459,39 @@ function showPopupWindow() {
 
 showPopupWindow(); 
 
-} //end of for desktop version
\ No newline at end of file
+} //end of popup window for desktop version
+
+
+
+
+// Contact Form Validation
+
+const emailInput = document.getElementById('email-input');
+console.log(emailInput.value);
```

Comment:
Please, remove console.log statements after debugging. so that your code will be more concise.